dropwizard-metrics/src/main/java/io/dropwizard/metrics/MetricsFactory.java

```diff
@@ -109,7 +103,10 @@ public void setReportOnStop(boolean reportOnStop) {
      *
      * @param environment the lifecycle to manage the reporters.
      * @param registry the metric registry to report metrics from.
+     *
+     * @since 2.0
      */
+    @JsonProperty
```

Comment:
I don’t think this should have an annotation.

Reply:
Right you are - looks like an extra `@JsonProperty` snuck in in eedb84c27cd3efb073eac959710ec59147e17a8b